site/content/integrate/apps/deployment/_index.md

```diff
@@ -52,40 +49,128 @@ For a go app the manifest snippet would look like this:
         }
     ]
 }
-``` 
+```
+
+### Setting up your Mattermost instance and AWS for deploying custom apps
```

Comment:
The previous two sections `App Bundle` and `Making your app runnable as an AWS Lambda function` are targeting devs while this section targets admins. Can we make that more clear? Maybe even split into multiple pages? Like one page for AWS and one (small) for HTTP.

Reply:
@hanzei, Please see [`86a9c95` (#826)](https://github.com/mattermost/mattermost-developer-documentation/pull/826/commits/86a9c953c776a9ded084e54a1eeeec23a623225f)

I created the following L2 Headers 
* Preparing your app for AWS (For App Developers)
* Deployment to AWS (For Admins) 

You have a valid point about separation of instruction into pages, which would benefit the organization of this doc.  

I would propose. 
* Deployment
  * AWS 
    * Preparing your app for AWS (For App Developers) 
    * Deployment to AWS (For Admins) 
    * Optional: Provision the hello-lambda app
    * Provisioned app details 
    * Provisioning in Mattermost Cloud 
  * HTTP Server
* Submit to Marketplace (this should not be in the deploying apps section, but it's own top-level Apps section (under Apps Dev Preview)

Reply:
What do you think?  I can make these changes easily.  I am out next week, but can get these in if you and @DHaussermann think it beneficial.  I think that would help separate and better define the main portions of the deploy process.